packages/client/src/lib/chess/core/game.test.ts

```diff
@@ -0,0 +1,362 @@
+  })
+
+  test('Стартовая расстановка танков [5]', () => {
+    expect((game.board.grid[0][13] as TankCell).data.type).toBe('HT')
+  })
+
+  test('Стартовая расстановка танков [6]', () => {
+    expect((game.board.grid[2][3] as TankCell).data.type).toBe('LT')
+  })
+
+  test('Стартовая расстановка танков [7]', () => {
+    expect((game.board.grid[2][6] as TankCell).data.type).toBe('LT')
+  })
+
+  test('Стартовая расстановка танков [8]', () => {
+    expect((game.board.grid[2][8] as TankCell).data.type).toBe('MT')
+  })
+
+  test('Стартовая расстановка танков [9]', () => {
+    expect((game.board.grid[2][10] as TankCell).data.type).toBe('LT')
+  })
+
+  test('Стартовая расстановка танков [10]', () => {
+    expect((game.board.grid[2][13] as TankCell).data.type).toBe('LT')
+  })
+
+  test('Стартовая расстановка танков [11]', () => {
+    expect((game.board.grid[13][2] as TankCell).data.type).toBe('LT')
+  })
+
+  test('Стартовая расстановка танков [12]', () => {
+    expect((game.board.grid[13][5] as TankCell).data.type).toBe('LT')
+  })
+
+  test('Стартовая расстановка танков [13]', () => {
+    expect((game.board.grid[13][7] as TankCell).data.type).toBe('MT')
+  })
+
+  test('Стартовая расстановка танков [14]', () => {
+    expect((game.board.grid[13][9] as TankCell).data.type).toBe('LT')
+  })
+
+  test('Стартовая расстановка танков [15]', () => {
+    expect((game.board.grid[13][12] as TankCell).data.type).toBe('LT')
+  })
+
+  test('Стартовая расстановка танков [16]', () => {
+    expect((game.board.grid[15][2] as TankCell).data.type).toBe('HT')
+  })
+
+  test('Стартовая расстановка танков [17]', () => {
+    expect((game.board.grid[15][5] as TankCell).data.type).toBe('MT')
+  })
+
+  test('Стартовая расстановка танков [18]', () => {
+    expect((game.board.grid[15][7] as TankCell).data.type).toBe('CLT')
+  })
+
+  test('Стартовая расстановка танков [19]', () => {
+    expect((game.board.grid[15][9] as TankCell).data.type).toBe('MT')
+  })
+
+  test('Стартовая расстановка танков [20]', () => {
+    expect((game.board.grid[15][12] as TankCell).data.type).toBe('HT')
+  })
+})
+
+/**--------------------------------------------------------------- */
+
+describe('Тестирование действий', () => {
+  let game: Game
+  let gamers: {
+    playerName1: string
+    playerName2: string
+  }
+  let testWhiteTank: Tank, testBlackTank: Tank, testWhiteTankLine2: Tank
+
+  beforeEach(() => {
+    game = new Game()
+    gamers = {
+      playerName1: 'Иван Иванов',
+      playerName2: 'Катя Катина',
+    }
+    game.startGame(gamers)
+    testWhiteTank = game.tanks[10]
+    testBlackTank = game.tanks[0]
+    testWhiteTankLine2 = game.tanks[18]
+  })
+
+  test('Перед ходом танку не доступно действие "Завершить ход"', () => {
+    game.setActiveTank(testWhiteTank.id)
+    const availableActArr = game.getAvailableActions(game.activeTank!)
+    expect(availableActArr.indexOf('STOP' as ACTION_TYPE)).toBe(-1)
```

Comment:
А зачем `as`? Это же `enum`, который можно так использовать
```suggestion
    expect(availableActArr.indexOf(ACTION_TYPE.STOP)).toBe(-1)
```

Reply:
попроавила